src/Files.App.Storage/FtpStorage/FtpStorable.cs

```diff
@@ -5,32 +5,32 @@
 
 namespace Files.App.Storage.FtpStorage
 {
+	/// <inheritdoc cref="IStorable"/>
 	public abstract class FtpStorable : ILocatableStorable
 	{
 		/// <inheritdoc/>
-		public string Path { get; protected set; }
+		public string Id => string.Empty;
```

Comment:
fyi, I wouldn't do anything to this for now, as `Id` will be implemented [_**here**_](https://github.com/files-community/Files/pull/10477/commits/fcc9db77ba895cbe503b569d521bb73ab8a753d6). (And in other places as well)